rpc/src/rpc.rs

```diff
@@ -1791,8 +1791,10 @@ impl JsonRpcRequestProcessor {
                             }
                         }
                     }
+                    Err(StorageError::SignatureNotFound) => {}
                     Err(err) => {
-                        warn!("{:?}", err);
+                        warn!("Failed to query Bigtable: {:?}", err);
+                        return Err(RpcCustomError::LongTermStorageUnreachable.into());
```

Comment:
Now that `storage_bigtable/lib.rs` properly throws SignatureNotFound errors, we can be sure that when this arm is matched, it's because of a Bigtable connection error.